tests/test_modeling_tf_common.py

```diff
@@ -125,6 +126,25 @@ def _config_zero_init(config):
     return configs_no_init
 
 
+def _return_type_has_loss(model):
+    return_type = get_type_hints(model.call)
+    if "return" not in return_type:
+        return False
+    return_type = return_type["return"]
+    if hasattr(return_type, "__args__"):  # Awkward check for union because UnionType only turns up in 3.10
+        for type_annotation in return_type.__args__:
+            if inspect.isclass(type_annotation) and issubclass(type_annotation, ModelOutput):
+                field_names = [field.name for field in fields(type_annotation)]
+                if "loss" in field_names:
+                    return True
```

Comment:
You could escape earlier 🏃 
```suggestion
                for field_name in fields(type_annotation):
                    if field_name == "loss":
                        return True
```

Reply:
I think I'll leave this one as is - I'd have to rewrite it with `field_name.name` and it might get a little ugly!